PrebidMobile/Objc/PrebidMobileRendering/Networking/Parameters/PBMGeoLocationParameterBuilder.m

```diff
@@ -59,6 +59,10 @@ - (void)buildBidRequest:(PBMORTBBidRequest *)bidRequest {
     
     if (self.locationManager.coordinatesAreValid) {
         CLLocationCoordinate2D coordinates = self.locationManager.coordinates;
+        
+        // Rounds with the precision defined in Targeting or returns the original coordinates if precision is 0.
+        coordinates = [[Utils shared] roundWithCoordinates:coordinates precision:[[Targeting shared] locationPrecision]];
```

Comment:
Please use `roundWithCoordinates` right in the line of initialization of `coordinates` (line 61).